src/main/java/trycb/web/TenantUserController.java

```diff
@@ -108,9 +118,11 @@ public Object booked(@PathVariable("tenant") String tenant, @PathVariable("usern
             jwtService.verifyAuthenticationHeader(authentication, username);
             return ResponseEntity.ok(tenantUserService.getFlightsForUser(bucket, tenant, username));
         } catch (IllegalStateException e) {
+            LOGGER.error("Failed with invalid state exception", e);
             return ResponseEntity.status(HttpStatus.FORBIDDEN)
                     .body(new Error("Forbidden, you don't have access to this cart"));
         } catch (IllegalArgumentException e) {
+            LOGGER.error("Failed with invalid argument exception", e);
```

Comment:
minor q: why is an IllegalArgumentException thrown to indicate this? (not important, just curious)

Reply:
Probably was there as good practice to check if the user has passed an invalid argument to the endpoint.